src/includes/class-alma-wc-admin-form.php

```diff
@@ -248,7 +248,7 @@ private function init_fee_plans_fields( $default_settings ) {
 	/**
 	 * Init default plugin fields
 	 *
-	 * @param array $default_settings as default settings.
+	 * @param array $default_settings default settings.
```

Comment:
Because this "as" has no reason to exist.This is not the supposed comment pattern expected. I removed it here but "as" should be removed everywhere in comments.